src/main/java/mate/academy/spring/model/dto/request/MovieRequestDto.java

```diff
@@ -1,7 +1,14 @@
 package mate.academy.spring.model.dto.request;
 
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+
 public class MovieRequestDto {
+    @NotNull
+    @NotEmpty
```

Comment:
```suggestion
    @NotBlank
```

we can use @NotBlank here